src/content/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data.mdx

```diff
@@ -10,34 +10,60 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import dashboardsDashboardMenu from 'images/dashboards_screenshot-crop_dashboard-menu.webp'
-
-
-[Dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) is a powerful tool to visualize and track how your system is doing, and allows you to tackle issues and plan next steps. Learn to share all that information with your team, customers, or other stakeholders, using the following options to import, share and export dashboards, or add and share charts (or other content) in a dashboard.
+You can use [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) to visual and track how your system is doing, tackle issues, and plan next steps. Learn to share all that information with your team, customers, or other stakeholders with various import and export options.
```

Comment:
think it should be "to visualize" and "plan any further steps" or something like that?